torch/csrc/distributed/rpc/agent_utils.h

```diff
@@ -16,6 +16,15 @@ std::unordered_map<std::string, worker_id_t> collectNames(
     const std::string& selfName,
     const int worldSize);
 
+// Ranks in dynamic RPC groups will initially call into this to establish the
+// name-to-id mapping for the current peers in the group. The current rank will
+// put its own worker info in the store and discover all the ranks that came
+// before it/
+std::unordered_map<std::string, worker_id_t> collectCurrentNames(
+    ::c10d::PrefixStore store,
```

Comment:
curious, why this is a `PrefixStore` instead of `const c10::intrusive_ptr<::c10d::Store>&`?